<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEdtWire</class>
 <widget class="QWidget" name="FormEdtWire">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Марки проволоки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableView" name="tableViewWire">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxCored">
           <property name="text">
            <string>Порошковая</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditMark"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxNerj">
           <property name="text">
            <string>Нержавейка</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Описание</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Марка</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="plainTextEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxWire">
           <property name="text">
            <string>Выпускаемая</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Коэф. расхода</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="Viewer" name="tableViewKr"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Упаковка, штрих код</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="Viewer" name="tableViewPack"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QTableView</extends>
   <header>viewer.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditMark</tabstop>
  <tabstop>checkBoxNerj</tabstop>
  <tabstop>checkBoxCored</tabstop>
  <tabstop>checkBoxWire</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>tableViewPack</tabstop>
  <tabstop>tableViewWire</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
